src/System/IO/Resource/Linear/Internal.hs

```diff
@@ -201,7 +202,7 @@ unsafeRelease (UnsafeResource key _) = RIO (\st -> Linear.mask_ (releaseWith key
 unsafeAcquire ::
```

Comment:
Is the docstring calling this unsafe still correct given your new posture towards resources?

Reply:
I consider it is an unsafe function in the usual sense in Haskell: it has conditions for application which can't be proved by the compiler, hence are the responsibility of the programmer.

Though I must admit that `ResourceT` has the same function and considers it safe. So there is certainly a question.